### PR TITLE
Remove duplicated and unused ticketbuyer config options.

### LINKED ISSUE #467: ticket buyer config should not be a separate file
Instead we can move it into the main config.  For organization, options should be put under a new section e.g. `[Ticket Buyer Options]` instead of the top-level `[Application Options]`.

### PR DESCRIPTION
**Pull request by dajohi**
Fixes #467

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -418,6 +472,27 @@ func loadConfig() (*config, []string, error) {
 		cfg.EnableVoting = true
 	}
 
+	// Make sure the fee source type given is valid.
+	switch cfg.TBOpts.FeeSource {
+	case ticketbuyer.TicketFeeMean:
+	case ticketbuyer.TicketFeeMedian:
+	default:
+		str := "%s: Invalid fee source '%s'"
+		err := fmt.Errorf(str, "loadTicketBuyerConfig", cfg.TBOpts.FeeSource)
```
Comment (jrick) [inline config.go]: if this is supposed to be the function name of the caller, it's not right.

File: config.go [hunk 2]
```diff
@@ -418,6 +472,27 @@ func loadConfig() (*config, []string, error) {
...
+	}
+
+	// Make sure a valid average price mode is given.
+	switch cfg.TBOpts.AvgPriceMode {
+	case ticketbuyer.PriceTargetVWAP:
+	case ticketbuyer.PriceTargetPool:
+	case ticketbuyer.PriceTargetDual:
+	default:
+		str := "%s: Invalid average price mode '%s'"
+		err := fmt.Errorf(str, "loadTicketBuyerConfig", cfg.TBOpts.AvgPriceMode)
```
Comment (jrick) [inline config.go]: here too

File: config.go (hunk 1 above)
Comment (dajohi) [inline config.go]: fixed

File: config.go (hunk 2 above)
Comment (dajohi) [inline config.go]: fixed.

**Comment by jrick** [comment]
Looks good, only thing I think is missing is to add a warning when the old ticketbuyer config is still found.  We should inform the user about this so if they haven't yet, they know to migrate their settings over into the wallet config and remove the old file.

**Pull request merged**